CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.3.0-rc.2] - 2025-09-04
```

Comment:
We use YYYY-MM-DD here, meaning this date is in September.

Please change to 2025-04-08 and I'll make a release today.

Reply:
No time left today, will do it tomorrow. Please use 2025-04-09

Reply:
Fixed, thanks!